app/cafe.py

```diff
@@ -0,0 +1,21 @@
+from datetime import date
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor.keys():
+            raise NotVaccinatedError("NotVaccinatedError")
+        elif date.today() > visitor["vaccine"]["expiration_date"]:
+            raise OutdatedVaccineError("OutdatedVaccineError")
+        elif visitor.get("wearing_a_mask") in [None, False]:
```

Comment:
You can use `not visitor.get("wearing_a_mask")` instead of `visitor.get("wearing_a_mask") in [None, False]`